images/prow-tests/Dockerfile

```diff
@@ -183,15 +184,15 @@ COPY images/prow-tests/in-gvm-env.sh /usr/local/bin
 FROM golang:1.17 AS external-go-gets
 
 ARG KUBETEST2_VERSION=5e5d3e9eebc6a609aa428bd3e2a1c0c3566d5baf
-ARG KIND_VERSION=v0.12.0
+ARG KIND_VERSION=v0.14.0
 ARG KO_VERSION=v0.11.2
-ARG PROTOC_GEN_GO_VERSION=v1.26.0
-ARG PROTOC_GEN_GOGOFASTER_VERSION=v1.3.1
-ARG GOTESTSUM_VERSION=v1.7.0
+ARG PROTOC_GEN_GO_VERSION=v1.29.0
```

Comment:
Looks like latest is 1.28 https://pkg.go.dev/google.golang.org/protobuf/cmd/protoc-gen-go